lib/project/web-site-project.ts

```diff
@@ -22,7 +22,7 @@ export class MobileWebSiteProject extends frameworkProjectBaseLib.FrameworkProje
 	}
 
 	public get name(): string {
-		return this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.MobileWebsite;
+		throw new Error("Тhe development of mobile websites is discontinued. For more information, see http://www.telerik.com/blogs/whats-next-for-telerik-platform-and-web-projects");
```

Comment:
`this.$errors.failWithoutHelp` maybe. Also you extract the message to separate variable/function.